### PR TITLE
All-in-one alternative for macOS for beginners

### PR DESCRIPTION
**Pull request by nathanfallet**
Beginners wanting to try OCaml might not be comfortable with a terminal to install OCaml from Homebrew or any package manager. That's why I added a paragraph with an All-in-one package to download from the App Store to easily start without being afraid of the terminal (which is easy to use when we know it, but not very user friendly for new comers)

### CODE REVIEW

**Comment by johnwhitington** [comment]
I have downloaded and installed this, and it works pretty well. It seems roughly to offer the same functionality as the TryOCaml website.

I think we should mention it in places where we currently mention TryOCaml. For example, in the "A First Hour with OCaml" tutorial.

This PR puts it in the "Install" page and the "Up and Running with OCaml" page.

It doesn't belong on the "Install" page, because it doesn't install OCaml. It doesn't really belong on "Up and Running with OCaml" which gives a set of concrete steps. If the user chooses this software instead, the rest of the "Up and Running" tutorial doesn't make sense.

**Comment by nathanfallet** [comment]
@johnwhitington It's not really the same thing as TryOCaml. This editor works offline, is installed on the computer, and has file management (open/save/edit multiple files at once), and much more things.

It's free and open source, so it will be updated and improved day by day. For example in the next update I will add a helper to install a native version of OCaml on the system (which will install Homebrew and OCaml automatically if needed), to use it instead of the built-in version made with jsoo. And many more things are planned to make it a better and more powerful editor.

**Review by ZirconiumZr40 — APPROVED** [review]

**Comment by johnwhitington** [comment]
> @johnwhitington It's not really the same thing as TryOCaml. This editor works offline, is installed on the computer, and has file management (open/save/edit multiple files at once), and much more things.
> 
> It's free and open source, so it will be updated and improved day by day. For example in the next update I will add a helper to install a native version of OCaml on the system (which will install Homebrew and OCaml automatically if needed), to use it instead of the built-in version made with jsoo. And many more things are planned to make it a better and more powerful editor.

All this could be true, in the future. And I hope it is.

It still doesn't, today, belong in "Up and Running with OCaml" or "Install" as done in this PR, for the reasons carefully explained above. It belongs in "A first hour with OCaml". In fact, not only does it belong there, it fits really nicely there.

**Comment by nathanfallet** [comment]
@johnwhitington Just updated my PR, you can review it again

**Comment by johnwhitington** [comment]
Looks good, thanks.

File: site/learn/tutorials/a_first_hour_with_ocaml.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ as described in [Up and Running](up_and_running.html).
 Alternatively, you may follow almost all of it by running OCaml in your browser
 using [TryOCaml](http://try.ocamlpro.com), with no installation required.
 
+On macOS/iOS/iPadOS, you can download [this All-in-one package on the App Store](https://apps.apple.com/app/ocaml-learn-code/id1547506826). It contains an editor side by side with an interactive top level, is free and [open source](https://github.com/GroupeMINASTE/OCaml-iOS).
```
Comment (avsm) [inline site/learn/tutorials/a_first_hour_with_ocaml.md]:
```suggestion
On macOS/iOS/iPadOS, you can download this [all-in-one package on the App Store](https://apps.apple.com/app/ocaml-learn-code/id1547506826). It contains an editor side-by-side with an interactive top level, and is free and [open source](https://github.com/GroupeMINASTE/OCaml-iOS).
```

**Comment by avsm** [comment]
Looks good to me in the new location, and merging with language tweaks. Thanks!

**Pull request merged**